WordPress/Classes/ViewRelated/Post/PostListConflictResolver.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+
+struct PostListConflictResolver {
+    /// Presents an alert with an option to discard the local or web version of a post, and opens the editor with the non-discarded version
+    static func handle(post: Post, in postListViewController: PostListViewController) {
+        let conflictResolutionAlert = presentConflictResolutionAlert(for: post) { discardWeb in
+            guard ReachabilityUtils.isInternetReachable() else {
```

Comment:
I don't think we need to handle offline support yet in this task.

Anyway, one of the principles that we follow is to always try to execute the request instead of relying only on the `ReachabilityUtils`. You can see that in action in `PostCordinator.dispatchNotice()`.

Reply:
Will remove, thanks for background!